### ISSUE TITLE
fast speech  mel normalization

### ISSUE THREAD

**Issue by superhg2012** [dathudeptrai/TensorflowTTS#35]
when compute mean and scaler for mel-spectrogram before normization,   mean and scaler are computed from all dataset and only the first frame mel?

` mel = mel[0].numpy()`

**Comment by dathudeptrai** [comment]
@superhg2012 mean, var for all TRAINING SET.

**Comment by superhg2012** [comment]
All right, I get it, thanks! By the way replace relu activation in ffn layer of FFT block with mish activation function really works for me.

**Comment by dathudeptrai** [comment]
@superhg2012 Fastspeech without duration from teacher model is comming :))) let see

**Comment by superhg2012** [comment]
> @superhg2012 Fastspeech without duration from teacher model is comming :))) let see

any paper reference?

**Comment by dathudeptrai** [comment]
@superhg2012 flow-tts, glow-tts

**Comment by dathudeptrai** [comment]
@superhg2012 i just added fastspeech samples, can you take a look and see if it's good or not :D. https://dathudeptrai.github.io/TensorflowTTS/
